spotify_assistant.yml

```diff
@@ -8,3 +8,5 @@ dependencies:
   - pandas
   - requests
   - spotipy
+  - pytest
+prefix: /Users/JulieS/opt/miniconda3/envs/spotifypy
```

Comment:
I think the prefix line should be removed, otherwise this is good

Reply:
Thanks Caroline